sauna/src/index.js

```diff
@@ -3,48 +3,42 @@
  * Track the time you spent coding, with a cool leaderboard.
  */
 import express from 'express';
-import mongoose from 'mongoose';
-import config from '../config.js';
-import bearer from 'express-bearer-token'; 
-import fs from 'fs';
-import path from 'path';
-
-console.log('Connecting to the mongoDB');
-try {
-    await mongoose.connect(config.mongodb_connection_string, {
-        useNewUrlParser: true,
-        useUnifiedTopology: true,
-        useFindAndModify: false,
-        useCreateIndex: true
+import bearer from 'express-bearer-token';
+import dotenv from 'dotenv';
+
+import usersController from './controllers/users.js';
+import heartbeatController from './controllers/heartbeat.js';
+
+const port = 8080;
```

Comment:
Because sauna is also serving React content, it would overall make more sense to use the port 80.

Reply:
It would make even more sense to add the port to environmental variables or config.